coap/Kconfig

```diff
@@ -102,6 +102,12 @@ menu "CoAP Configuration"
 
             If this option is disabled, redundant CoAP Observe Persist code is removed.
 
+    config COAP_Q_BLOCK
+        bool "Enable block-wise transfer in CoAP"
```

Comment:
bool "Enable Q-Block (RFC9177) support in CoAP"

Note: change as regular block transfers (RFC7959) is always enabled

Reply:
Change is good